src/BloomExe/Browser.cs

```diff
@@ -122,6 +122,13 @@ public static void SetUpXulRunner()
 			GeckoPreferences.User["network.proxy.http"] = string.Empty;
 			GeckoPreferences.User["network.proxy.http_port"] = 80;
 			GeckoPreferences.User["network.proxy.type"] = 1; // 0 = direct (uses system settings on Windows), 1 = manual configuration
+			// Try some settings to reduce memory consumption by the mozilla browser engine.
+			// See: http://forums.macrumors.com/showthread.php?t=1838393 and about:config (in firefox)
+			GeckoPreferences.User["memory.free_dirty_pages"] = true;
+			GeckoPreferences.User["browser.sessionhistory.max_entries"] = 1;
+			GeckoPreferences.User["browser.sessionhistory.max_total_viewers"] = 0;
+			GeckoPreferences.User["browser.sessionstore.max_tabs_undo"] = 0;
```

Comment:
You've probably checked this...but it's fairly common to make 0 mean 'unlimited', which you probably don't want here.